### PR TITLE
feat(route): add Peking Univ cls announcement

### PR DESCRIPTION
**Pull request by william-swl**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/pku/cls/announcement
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Documentation / 文档说明
  - [ ] EN / 英文文档
  - [x] CN / 中文文档
- [x] Full text / 全文获取
  - [x] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制 
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施? 
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/pku/cls/announcement.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                const $ = cheerio.load(response);
+
+                item.description = $('div.page div.common_width div.col-md-9').first().html();
+                return item;
+            })
+        )
+    );
+
+    ctx.state.data = {
+        title: `北京大学生命科学学院通知公告`,
```
Comment (TonyRL) [inline lib/v2/pku/cls/announcement.js:38]:
```suggestion
        title: '北京大学生命科学学院通知公告',
```

File: lib/v2/pku/cls/announcement.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+const homeUrl = 'http://bio.pku.edu.cn/homes/Index/news/21/21.html';
+
+const baseUrl = 'http://bio.pku.edu.cn';
```
Comment (TonyRL) [inline lib/v2/pku/cls/announcement.js:7]:
```suggestion
const homeUrl = 'https://bio.pku.edu.cn/homes/Index/news/21/21.html';

const baseUrl = 'https://bio.pku.edu.cn';
```

File: lib/v2/pku/router.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ module.exports = function (router) {
     router.get('/admission/sszs', require('./pkuyjs'));
     router.get('/bbs/hot', require('./bbs/hot'));
     router.get('/cls/lecture', require('./cls/lecture'));
+    router.get('/cls/announcement', require('./cls/announcement'));
```
Comment (TonyRL) [inline lib/v2/pku/router.js:5]:
```suggestion
    router.get('/cls/announcement', require('./cls/announcement'));
    router.get('/cls/lecture', require('./cls/lecture'));
```

**Review by TonyRL — COMMENTED** [review]
Missing an entry in `maintainer.js`

[docs](https://docs.rsshub.app/joinus/new-rss/before-start#maintainerjs)

**Review by william-swl — COMMENTED** [review]
update

**Pull request merged**